Plugins/Public/playercntl_plugin/PimpShip.cpp

```diff
@@ -192,58 +208,35 @@ namespace PimpShip
 		if (!set_bEnablePimpShip)
 			return false;
 
-		uint iLocationID = 0;
-		pub::Player::GetLocation(iClientID, iLocationID);
-		if (set_mapDealers.find(iLocationID)==set_mapDealers.end())
+		if (!mapInfo[iClientID].bInPimpDealer)
 		{
-			uint iBaseID = 0;
-			pub::Player::GetBase(iClientID, iBaseID);
-			if (set_mapDealers.find(iBaseID)==set_mapDealers.end())
-			{
-				mapInfo[iClientID].bInPimpDealer = false;
-				mapInfo[iClientID].mapCurrEquip.clear();
-				PrintUserCmdText(iClientID, L"ERR ship pimping facilities not available at this base.");
-				return false;
-			}
+			PrintUserCmdText(iClientID, L"ERR Pimpship facilities are not available here");
+			return true;
 		}
 
-		mapInfo[iClientID].mapCurrEquip.clear();
-		mapInfo[iClientID].bInPimpDealer = true;
-
 		PrintUserCmdText(iClientID, L"Available ship pimping commands:");
+		PrintUserCmdText(iClientID, L"This facility costs " + ToMoneyStr(set_iCost) + L" credits to use per one item.");
 
 		PrintUserCmdText(iClientID, L"/showsetup");
 		PrintUserCmdText(iClientID, L"|     Display current ship setup.");
 
-		PrintUserCmdText(iClientID, L"/showitems");
+		PrintUserCmdText(iClientID, L"/showitems [from]-[to]");
 		PrintUserCmdText(iClientID, L"|     Display items that may be added to your ship.");
 
-		PrintUserCmdText(iClientID, L"/setitem <hardpoint id> <new item id>");
-		PrintUserCmdText(iClientID, L"|     Change the item at <hp id> to <item id>.");
-		PrintUserCmdText(iClientID, L"|     <hi id>s are shown by typing /show setup.");
+		PrintUserCmdText(iClientID, L"/setitem <hardpoint id> <item id>");
+		PrintUserCmdText(iClientID, L"|     Change the item at <hardpoint id> to <item id>.");
+		PrintUserCmdText(iClientID, L"|     <hardpoint id>s are shown by typing /show setup.");
 		PrintUserCmdText(iClientID, L"|     <item id>s are shown by typing /show items.");
+		PrintUserCmdText(iClientID, L"|     Allowed to print item name instead of ID.");
```

Comment:
Let's use 'item' instead of 'item id' then, and say it can be name or ID shown on /show items?